astroquery/skyview/core.py

```diff
@@ -79,16 +79,18 @@ def _generate_payload(self, input=None):
         url = urlparse.urljoin(self.URL, form.get('action'))
         return url, payload
 
-    def _submit_form(self, input=None, cache=True):
+    def _submit_form(self, input=None, cache=True, get_query_payload=False):
         url, payload = self._generate_payload(input=input)
+        if get_query_payload:
+            return payload
         response = self._request('GET', url, params=payload, cache=cache)
         response.raise_for_status()
         return response
 
     def get_images(self, position, survey, *, coordinates=None, projection=None,
                    pixels=None, scaling=None, sampler=None, resolver=None,
                    deedger=None, radius=None, height=None, width=None, cache=True,
-                   show_progress=True):
+                   show_progress=True, get_query_payload=False, **kwargs):
```

Comment:
do not add **kwargs here
```suggestion
                   show_progress=True, get_query_payload=False):
```